doc/standards-object.md

```diff
@@ -0,0 +1,139 @@
+- `allowedAttrs` - array(optional). List of allowed attributes (besides any required and global ARIA attributes).
+- `nameFromContent` - boolean(optional. Default `false`). If the role allows name from content when calculating the accessible name.
+- `unsupported` - boolean(optional. Default `false`). If the role role is unsupported. Use this property to disable a role.
+
+## Dpub Roles
+
+Dpub roles are defined in the [dpub-roles](../lib/standards/dpub-roles.js) file. Their structure is the exact same as the aria roles standard (albeit they have different `type`s) and is combined into the `ariaRoles` table. They are only separated for organizational purposes.
+
+## HTML Elms
+
+The [`htmlElms`](../lib/standards/html-elms.js) object defines valid HTML elements, their content type, and if they are allowed ARIA roles or attributes.
+
+### Used by Rules
+
+- `aria-allowed-attr` - Checks if the attribute can be used on the element from the `noAriaAttrs` property.
+- `aria-allowed-role` - Checks if the role can be used on the HTML element from the `allowedRoles` property.
+- `aria-required-attrs` - Checks if any required attrs are defied implicitly on the element from the `implicitAttrs` property.
+
+### Structure
+
+- `contentTypes` - array(required). List of [content models](https://html.spec.whatwg.org/multipage/dom.html#content-models) for the element. Valid values are:
+  - `flow`
+  - `sectioning`
+  - `heading`
+  - `phrasing`
+  - `embedded`
+  - `interactive`
+- `allowedRoles` - boolean or array(required). If element is allowed to use ARIA roles. A value of `true` means any role while a list of roles means only those are allowed. A value of `false` means no roles are allowed.
+- `noAriaAttrs` - boolean(optional. Defaults `true`). If the element is allowed to use global ARIA attributes and any allowed for the elements role.
+- `shadowRoot` - boolean(optional. Default `false`). If the element is allowed to have a shadow root.
+- `implicitAttrs` - object(optional. Default `{}`). Any implicit ARIA attributes for the element and their default value.
+- `namingMethods` - array(optional. Default `[]`). The [native text method](../lib/commons/text/native-text-methods.js) used to calculate the accessible name of the element.
+- `variant` - object(optional). Sometimes an element can have different properties based on which attributes are defined on it. For example, an `img` is only considered interactive if it has the `usemap` attribute.
+
+  For these cases, the `variant` property allows different definitions for the element. The `variant` object gives a name to each variant, defines when the variant applies through the `matches` property (a `matcher` object), and provides a `default` variant in the case none of the other variants match. A variant should only define what can change based on the attributes. Any properties that always apply regardless of attributes should be defined at the top level.
```

Comment:
Should we have a separate section for Variant that explains this, instead of having it all be part of this listitem?